crates/fluvio/src/lib.rs

```diff
@@ -46,6 +46,7 @@ use tracing::instrument;
 
 /// The minimum VERSION of the Fluvio Platform that this client is compatible with.
 const MINIMUM_PLATFORM_VERSION: &str = "0.9.0";
+pub(crate) const VERSION: &str = include_str!("../../../VERSION");
```

Comment:
did we have this somewhere already?   Seems like we should name this PLATFORM_VERSION

Reply:
Yeah, we have this same const in many places: 
<img width="894" alt="image" src="https://github.com/user-attachments/assets/aa403127-8279-4229-87b5-1b024156bbf4">

Reply:
Sure, rename only in this case?

Reply:
About `MINIMUM_PLATFORM_VERSION` it's been checked after the lookup_version of the watch API. And will return it: 

```bash
    #[error("Cluster (with platform version {cluster_version}) is older than the minimum required version {client_minimum_version}
To interact with this cluster, please install the matching CLI version using the following command:
    curl -fsS https://hub.infinyon.cloud/install/install.sh | VERSION={cluster_version} bash
    ")]
```

But as we're not updating this const does not seem so useful.

Reply:
Should I remove it?

Reply:
Hmm, let's just leave it all alone until we get a improved wider compatability check.  

Lets leave leaving the constant as VERSION.   Forget renaming it  PLATFORM_VERSION as it would be out of place with all the other crates.